custom/conf/app.example.ini

```diff
@@ -682,6 +682,29 @@ LEVEL = Info
 ;; Disable the usage of using partial clones for git.
 ;DISABLE_PARTIAL_CLONE = false
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Git Operation timeout in seconds
+;[git.timeout]
+;DEFAULT = 360
+;MIGRATE = 600
+;MIRROR = 300
+;CLONE = 300
+;PULL = 300
+;GC = 60
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Git Reflog timeout in days
+;[git.reflog]
+;ENABLED = true
+;EXPIRATION = 90
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Git config options
+;; This section only does "set" config, a removed config key from this section won't be removed from git config automatically
+;[git.config]
+;diff.algorithm = histogram
+;other.config-key = ...
```

Comment:
This example is the not default, would remove. You can outline the example in the comment above instead.

Reply:
Can you provide a suggested change?

Reply:
Done below.